integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/datamap/CarbonDataMapShowCommand.scala

```diff
@@ -66,11 +67,24 @@ case class CarbonDataMapShowCommand(tableIdentifier: Option[TableIdentifier])
 
   private def convertToRow(schemaList: util.List[DataMapSchema]) = {
     if (schemaList != null && schemaList.size() > 0) {
-      schemaList.asScala.map { s =>
-        var table = "(NA)"
-        val relationIdentifier = s.getRelationIdentifier
-          table = relationIdentifier.getDatabaseName + "." + relationIdentifier.getTableName
-        Row(s.getDataMapName, s.getProviderName, table)
+      schemaList.asScala
+        .map { s =>
+          val relationIdentifier = s.getRelationIdentifier
+          val table = relationIdentifier.getDatabaseName + "." + relationIdentifier.getTableName
+          // preaggregate datamap does not support user specified property, therefor we return empty
+          val dmPropertieStr = if (s.getProviderName.equalsIgnoreCase(
+              DataMapClassProvider.PREAGGREGATE.getShortName)) {
```

Comment:
How about timeseries and MV datamap?

Reply:
Timeseries datamap is OK, please refer to the added test.

MV datamap currently is not available in this test case. When I added MV related test, the test will fail which complains that it cannot find MV